tests/test_bundle_download.py

```diff
@@ -82,7 +82,19 @@ def test_download_bundle(self, bundle_name, version):
                     if file == "network.json":
                         self.assertTrue(check_hash(filepath=file_path, val=hash_val))
 
-    @parameterized.expand([TEST_CASE_4])
+    @parameterized.expand([TEST_CASE_NGC_1])
+    @skip_if_quick
+    def test_ngc_download_bundle(self, bundle_name, version, file_path, hash_val):
+        with skip_if_downloading_fails():
+            with tempfile.TemporaryDirectory() as tempdir:
+                cmd = ["coverage", "run", "-m", "monai.bundle", "download", "--name", bundle_name, "--source", "ngc"]
+                cmd += ["--bundle_dir", tempdir, "--progress", "False", "--version", version]
+                command_line_tests(cmd)
```

Comment:
this will run many times everyday for different CI jobs, it might give us a wrong impression about the ngc model's number of downloadings...

how about we add it to this file
https://github.com/Project-MONAI/MONAI/blob/dev/tests/ngc_mmar_loading.py

and re-enable the test here (once per day):
https://github.com/Project-MONAI/MONAI/blob/e2434fdd9f51d2bf3dc29193f8cdc756a9aa2e3f/.github/workflows/cron-mmar.yml#L6

Reply:
thanks @wyli , let me check it.

Reply:
Uncomment here to enable the daily schedule https://github.com/Project-MONAI/MONAI/blob/0abd04e9cd1fd955d48501fe7a0c149f5fad86cf/.github/workflows/cron-mmar.yml#L6

Reply:
Thanks @wyli . BTW, should the name be changed to cron-bundle?

Reply:
and for the test file, can it be `ngc_bundle_download.py`?

Reply:
sure, please rename all the mmar related